### PR TITLE
Jws revised issue1617

### LINKED ISSUE #1617: Add the head information when plotting the gifures.
For each figure, add the head information (measure name or group name).

### PR DESCRIPTION
**Pull request by JiaweiSun0512**

### CODE REVIEW

File: braph2genesis/src/analysis/_AnalyzeGroupPP_G.gen.m [hunk 1]
```diff
@@ -667,7 +672,7 @@ function cb_open_plots(~, ~)
         end
 
         gui_pfm = gui.get('PE').get('PR_DICT').get('IT', 'PFM').memorize('GUI_ITEM');
-    
+        set(gui_pfm, 'TITLE', ['Line plot: ' measure_fullname]);
```
Comment (giovannivolpe) [inline braph2genesis/src/analysis/_AnalyzeGroupPP_G.gen.m]: use m.get('NAME') instead

File: braph2genesis/src/analysis/_AnalyzeGroupPP_G.gen.m [hunk 2]
```diff
@@ -585,6 +587,7 @@ function cb_open_mbrain(~, ~)
             end
     
             gui = GUIFig( ...
+                'TITLE', ['Brain plot: ' measure_fullname], ...
```
Comment (giovannivolpe) [inline braph2genesis/src/analysis/_AnalyzeGroupPP_G.gen.m]: use m.get('NAME')

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
use m.get('NAME') instead of getting the file name

**Review by giovannivolpe — CHANGES_REQUESTED** [review]
In the case of the graph, it should be the graph LABEL prop, not the g_key (which the users cannot change)

**Review by bzufiriaki — APPROVED** [review]

**Pull request merged**